src/oca_github_bot/tasks/migration_issue_bot.py

```diff
@@ -29,6 +29,17 @@ def _find_issue(gh_repo, milestone, target_branch):
     return issue
 
 
+def _check_line_issue(gh_pr_number, issue_body):
+    lines = []
+    regex = r"\#%s\b" % gh_pr_number
+    for line in issue_body.split("\n"):
+        if re.findall(regex, line):
+            lines.append(line[:3] + "x" + line[4:])
```

Comment:
Thanks for the PR!
```suggestion
            checked_line = line.replace('[ ]', '[x]', 1)
            lines.append(checked_line)
```
I think the suggested change is safer if the line is not exactly what has been created by https://github.com/OCA/oca-github-bot/blob/c8555ba1cd604028d511b29cfa65010dcf7c8403/src/oca_github_bot/tasks/migration_issue_bot.py#L32

For instance, in https://github.com/OCA/l10n-italy/issues/1905 we have indented the lines to highlight dependencies.
Regex might be used better but I think this is already a good start :)